lib/product-import.coffee

```diff
@@ -168,7 +168,6 @@ class ProductImport
         @client.productProjections
         .where(predicate)
         .staged(true)
-        .perPage(100)
```

Comment:
Combination of `.perPage(..).all()` was throwing an error:
```
Query limit doesn't seem to be 0. This function queries all results, are you sure you want to use this?
```
This should be changed now by this commit but still - why we need `perPage()` when we use `all()`?